### PR TITLE
reduce re-renders on users and groups

### PR DESCRIPTION
**Pull request by Sbonelo01**
Related issues: [RENDER tidy]

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have merged the develop branch into my branch and fixed any merge conflicts
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by TheMaja — APPROVED** [review]

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Please address the actual issue. This code doesn't prevent unnecessary re-renders at all.

Reviewers, please review properly.

**Review by KhalidUmuzi — APPROVED** [review]

File: frontend/src/components/widgets/ActionLogCards.js [hunk 1]
```diff
@@ -35,14 +35,12 @@ const useStyles = makeStyles((theme) => ({
     variant: "outlined",
     color: "default",
     size: "small",
-    ["@media (max-width:620px)"]: {
-      // eslint-disable-line no-useless-computed-key
+    "@media (max-width:620px)": {
```
Comment (sheenarbw) [inline frontend/src/components/widgets/ActionLogCards.js]: Why did you remove the comments? Do you think the es-lint warnings should be there?  Sometimes it is good to suppress certain warnings

**Review by sheenarbw — CHANGES_REQUESTED** [review]
I was getting some really confusing errors when trying to run this. Please make sure your code is up to date with develop. Either merge develop into your branch, or rebase your branch onto develop. 

There are also merge conflicts.

And there are some comments you removed that should be put back. Did you remove them on purpose?

File: frontend/src/components/widgets/ActionLogCards.js (hunk 1 above)
Comment (Sbonelo01) [reply frontend/src/components/widgets/ActionLogCards.js]: I removed the square brackets and that resolved the ES Lint warnings. https://eslint.org/docs/latest/rules/no-useless-computed-key

So I did not think the comments were still nessesary.

File: frontend/src/components/widgets/ActionLogCards.js [hunk 2]
```diff
@@ -35,14 +35,12 @@ const useStyles = makeStyles((theme) => ({
     variant: "outlined",
     color: "default",
     size: "small",
-    
-    ["@media (max-width:620px)"]: {// eslint-disable-line no-useless-computed-key
+    "@media (max-width:620px)":
       fontSize: "0.7rem",
```
Comment (KhalidUmuzi) [inline frontend/src/components/widgets/ActionLogCards.js]: This line gives us an error when trying to run it. I don't think this works and I think we need to revert back to having the ESLint warnings and deal with it

**Review by KhalidUmuzi — CHANGES_REQUESTED** [review]

**Comment by KhalidUmuzi** [comment]
Merge conflicts

**Comment by KhalidUmuzi** [comment]
There seems to still be conflicts on this one

frontend/src/components/pages/UsersAndGroups/index.jsx

**Review by KhalidUmuzi — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]
